Dockerfile

```
@@ -0,0 +1,47 @@
+
+# Set working directory
+WORKDIR /app
+
+# Copy built application from builder stage
+COPY --from=builder /app/dist ./dist
+COPY --from=builder /app/node_modules ./node_modules
+COPY --from=builder /app/package*.json ./
+COPY --from=builder /app/prisma ./prisma
+
+# Install only Prisma CLI
+RUN npm install prisma --no-save && npx prisma generate && npm cache clean --force
```

Hadolint (2.12.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)